smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/ImportDeclarations.java

```diff
@@ -63,34 +69,41 @@ ImportDeclarations addImport(String name, String alias, String module) {
     public String toString() {
         StringBuilder result = new StringBuilder();
 
-        if (!imports.isEmpty()) {
-            for (Map.Entry<String, Map<String, String>> entry : imports.entrySet()) {
+        if (!defaultImports.isEmpty()) {
+            List<Map.Entry<String, String>> entries = new ArrayList<>(defaultImports.entrySet());
+            for (Map.Entry<String, String> importEntry : entries) {
```

Comment:
You can get rid of the List on the previous line and just say:

```
for (Map.Entry<String, String> importEntry : defaultImports.entrySet()) {
```

Reply:
Updated in 79a181d6fb006b740c8257223df1f06e11252e08